api/models/coordinator.py

```diff
@@ -37,6 +37,7 @@ def filter_active(self) -> "CoordinatorQuerySet":
 class Coordinator(Person):
     """Model for a coordinator."""
 
+    legacy_cid = models.IntegerField(null=True)  # from the old database
```

Comment:
I think that this comment can be put into `help_text`. It will be used both as a comment here and as a hint in Django admin